client/src/components/NavBarLogin.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+
+const NavBarLogin = props => {
+  const { account, classes, setLoggedOutAccount } = props;
+
+  const loginLink = (
+    <li className={classes.userLogin}>
+      <Link className={`${classes.link} ${classes.lastItem}`} to="/login">
+        Login
+      </Link>
+    </li>
+  );
+
+  const userGreeting = (
```

Comment:
Maybe convert this value into a function so it will run only when called
```suggestion
  const getUserGreeting = (account) => (
```